sympy/printing/tests/test_codeprinter.py

```diff
@@ -38,11 +38,19 @@ def test_print_Symbol():
     assert p._print(y) == 'if_He_Man'
 
 def test_issue_15791():
```

Comment:
This wasnt a good test of gh-15791. This PR is currently a regression, will update with a fix.